apollo-federation/src/sources/connect/expand/tests/schemas/steelthread.graphql

```diff
@@ -1,6 +1,7 @@
 schema
   @link(url: "https://specs.apollo.dev/link/v1.0")
-  @link(url: "https://specs.apollo.dev/join/v0.4", for: EXECUTION)
+  @link(url: "https://specs.apollo.dev/join/v0.5", for: EXECUTION)
```

Comment:
Not really a comment on the impl, but is there a reason that our docs for join stop after `v0.3`?

[This link](https://specs.apollo.dev/join/v0.5/) results in a 404 :(

Reply:
yeah we need to keep these up to date better. they're very much internal tho